Src/BootCamp.Chapter/Program.cs

```diff
@@ -8,14 +8,14 @@ static void Main(string[] args)
         {
 
             //Set variable
-            bool backtotop = true;
+            var backtotop = true;
             
 
```

Comment:
Those lines can be deleted because they do not affect your code in any way. If you enter or loop through the loop backtotop will be true so setting it here again will not do anything.

Reply:
Im confused, Dont I need to set backtotop to true so my while loop starts? otherwise I assumed it wouldn't start? 
also If I remove line 11 from my code I get an error on line 15 because "backtop" isn't defined as anything to be used in my while loop? 
error: 
Error	CS0103	The name 'backtotop' does not exist in the current context

Reply:
I am very sorry. I commented on the wrong line. I meant line 19! 🙈

Reply:
No worries, makes sense :) I have removed line 19. Thanks